src/containers/Tenant/Diagnostics/Healthcheck/IssuesViewer/IssueViewer.scss

```diff
@@ -9,18 +9,16 @@
     cursor: pointer;
 
     &__field {
-        padding: 0 10px;
+        display: flex;
+        justify-content: space-between;
```

Comment:
this doesn't make sense here, this is a property of a flex container, and `__field` is a flex item

well, it is a container since you made it one, but they both contain a single child, so they don't need to be